src/tools/rbd_mirror/Replayer.cc

```diff
@@ -311,7 +356,7 @@ int Replayer::init()
     // librados::Rados::conf_parse_argv
     r = cct->_conf->parse_argv(m_args);
```

Comment:
@dillaman Need to pass a copy of m_args, because parse_argv removes parsed items from the vector. You can see this running rbd_mirror.sh and observing that log and asok file names are not set properly set for the remote context.